src/core/sr.ml

```diff
@@ -123,9 +123,9 @@ let symb_to_tenv
   in
   symb_to_tenv t
 
-(** [check_rule r] checks whether the pre-rule [r] is well-typed in
-   signature state [ss] and then construct the corresponding rule. Note that
-   [Fatal] is raised in case of error. *)
+(** [check_rule pr] checks whether the pre-rule [pr] is well-typed in
+    signature state [ss] and then construct the corresponding rule.
+    Note that [Fatal] is raised in case of error. *)
```

Comment:
No need to break line

Reply:
You mean: add @raise?

Reply:
Ah yes, I did not mean this, but you're right.